api/docs/debugging.dox

```diff
@@ -78,12 +78,10 @@ the shared library that identifies the debug file.
 
 ## Expected Signals
 
-On Linux, DR sends itself a SIGILL signal during initialization, in order to measure the size of the signal frame used by the kernel.  If you are under gdb, simply continue past this signal.
+On Linux, DR sends itself a SIGILL signal during initialization, in order to measure the size of the signal frame used by the kernel.  If you are under gdb, simply continue past this signal.  SIGILL (SIGFPE on MacOS; SIGSTKFLT inside QEMU) is also sent to other threads to suspend or terminate them later on during execution.
```

Comment:
Now SIGILL is used also for attach right?

Reply:
See the next sentence "SIGILL...is also sent..."